<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_log</class>
 <widget class="QMainWindow" name="Main_log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Main_log {
border-image: url(:/icon/icon/background.jpg)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">centralwidget {
border-image: url(:/background/_1 (1).jpg);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>11</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="mainwidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>105</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>174</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QWidget" name="Container" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>675</width>
           <height>460</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">/* Áp dụng cho Container chính */
#Container {
    background-color: rgba(40, 40, 40, 180);
    border-radius: 15px;
    border: 1px solid rgba(60, 60, 60, 200);
    color: white;
}

/* Đặt lại thuộc tính cho các widget con */
QLineEdit {
    background-color: rgba(255, 255, 255, 220);
    color: black;
    border: 1px solid gray;
    border-radius: 5px;
    padding: 2px;
}

QPushButton {
    background-color: rgba(70, 130, 180, 200);
    color: white;
    border-radius: 5px;
}

QLabel {
    background-color: transparent;
    border: none;
}
</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>80</number>
          </property>
          <property name="topMargin">
           <number>30</number>
          </property>
          <property name="rightMargin">
           <number>80</number>
          </property>
          <property name="bottomMargin">
           <number>30</number>
          </property>
          <item row="0" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>31</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>9</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>16</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">
color: rgb(255,255,255); /* Đổi thành màu trắng */

</string>
              </property>
              <property name="text">
               <string>CHÀO MỪNG ĐẾN VỚI OFFICEHUB !</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLineEdit" name="username_line">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>350</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 0);
    border: 2px solid rgba(0, 0, 0, 0);
    border-bottom: 2px solid rgba(255, 255, 255, 200);
    color: rgb(255, 255, 255);
    padding-bottom: 7px;
    font-size: 14px;
}

QLineEdit:hover {
    border-bottom: 2px solid rgba(255, 255, 255, 255); /* Sáng hơn khi hover */
}

QLineEdit:focus {
    border-bottom: 2px solid rgba(0, 170, 255, 255); /* Đổi màu khi nhập */
    outline: none;
}
</string>
                </property>
                <property name="placeholderText">
                 <string>Tên đăng nhập...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>13</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="password_line">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 0);
    border: 2px solid rgba(0, 0, 0, 0);
    border-bottom: 2px solid rgba(255, 255, 255, 200);
    color: rgb(255, 255, 255);
    padding-bottom: 7px;
    font-size: 14px;
}

QLineEdit:hover {
    border-bottom: 2px solid rgba(255, 255, 255, 255); /* Sáng hơn khi hover */
}

QLineEdit:focus {
    border-bottom: 2px solid rgba(0, 170, 255, 255); /* Đổi màu khi nhập */
    outline: none;
}
</string>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Mật khẩu...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>13</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="login_btn">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: rgba(0, 0, 0, 0.6);
    color: white;
    border: 1px solid white;
    border-radius:20px;
    padding: 8px 16px;
    font-weight: bold;
    font-size: 11pt;
}

QPushButton:hover {
    background-color: rgba(255, 107, 53, 0.8);
    border: 1px solid #ff6b35;
}

QPushButton:pressed {
    background-color: rgba(255, 107, 53, 1);
    border: 1px solid white;
}
</string>
              </property>
              <property name="text">
               <string>Đ Ă N G  N H Ậ P</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="register_btn">
                <property name="contextMenuPolicy">
                 <enum>Qt::DefaultContextMenu</enum>
                </property>
                <property name="toolTipDuration">
                 <number>0</number>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel {
    color: rgb(255, 165, 0);
    text-decoration: none;
}

QLabel:hover {
    color: rgb(255, 255, 255); /* Màu trắng khi hover */
    text-decoration: underline;
}

QLabel:pressed {
    color: rgb(0, 170, 255); /* Màu xanh lam nhạt khi nhấn */
    text-decoration: underline;
}
</string>
                </property>
                <property name="text">
                 <string>Đăng ký ngay !</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>31</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>174</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>137</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="QLabel" name="label_18">
         <property name="minimumSize">
          <size>
           <width>1051</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(30, 30, 30);
</string>
         </property>
         <property name="text">
          <string>   © 2025 Le Huu Tri - Nguyen Minh Tu - Do Xuan Tri . All rights reserved.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
